doc/manual/src/design/store/store.md

```diff
@@ -1,10 +1,21 @@
-# Nix Store
+# Store
 
-A Nix store is a collection of *store objects* referred to by *store paths*.
-Every store also has a "store directory path", which is a path prefix used for various purposes.
+A Nix store is a collection of [store objects](objects.md) and associated operations.
```

Comment:
Technically this says a store "is a collection of...and...operations" but the operations are not collected I don't think?

How about?
```suggestion
A Nix store is a collection of [store objects](objects.md) that supports certain standard operations.
```

Reply:
If you say "certain", it has the opposite effect of being vague, as the word avoids saying explicitly what those are. If we say it explicitly afterwards, there is no point alluding to that these operations are specific.

I find "standard" problematic, because what does that standard refer to? Where is that standard defined? Since I know what you want to get at, I understand that you mean "these operations define the default store API", but that is not visible from the text. I would like to avoid wording it that way, because then we should just specify that API and not say "it has an API". Currently we're trying to give an informal overview, and give a prose specification in the other paragraph, and maybe at some point we should just add a class diagram for clarity?

How about that?
```suggestion
A Nix store is a collection of [store objects](objects.md) with associated operations.
```
It's semantically not exactly what you suggest, because now it sounds like the objects have operations. This is not wrong, therefore I'd be fine with it, but I see you wanted to say that the store (or the collection in its entirety) has operations.

See the other comment for another variant.